### PR TITLE
Adds first working graphql query

### PR DESCRIPTION
**Pull request by Vassyli**
This commit allows to query for realm information:
- coreVersion [string], fetches information from
  LotGD\Core\Game->gameVersion()
- crateVersion [string], static return value
- name [string], static return value

To run the server, use

```
php bin/console server:run
```

To write a realm query, call then the following url:

```
http://localhost:8000/?query=query%20RealmQuery{Realm{name,crateVersion,coreVersion}}
```

### CODE REVIEW

File: app/config/config.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
+imports:
+    - { resource: parameters.yml }
+    - { resource: security.yml }
+    - { resource: services.yml }
+
+# Put parameters here that don't need to change on each machine where the app is deployed
+# http://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
```
Comment (austenmc) [inline app/config/config.yml:7]: Not sure these fit this definition, but how crate/core config will be done is something we can iterate on.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+        "doctrine/doctrine-bundle": "^1.6",
+        "doctrine/doctrine-cache-bundle": "^1.2",
+        "symfony/swiftmailer-bundle": "^2.3",
+        "symfony/monolog-bundle": "^2.8",
+        "symfony/polyfill-apcu": "^1.0",
+        "sensio/distribution-bundle": "^5.0",
+        "sensio/framework-extra-bundle": "^3.0.2",
+        "incenteev/composer-parameter-handler": "^2.0",
+        
+        "lotgd/core": "dev-feature/buffs",
```
Comment (austenmc) [inline composer.json]: Let's try not to check this in. Set up it up to point to the lotgd/core repo on github w/ dev-master.

File: src/AppBundle/Resources/config/graphql/Realm.types.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# Notice about creation of types:
+#   for array return valies in the Resolver, use object.
+#   If the type is interface, it needs a resolveType argument to resolve the real type.
+Realm:
+    type: object
+    config:
+        description: "Common informations about the Realm"
```
Comment (austenmc) [inline src/AppBundle/Resources/config/graphql/Realm.types.yml]: "information" is plural already.

File: src/AppBundle/Resources/config/graphql/Realm.types.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        fields:
+            name:
+                type: "String"
+                description: "The name of this realm."
```
Comment (austenmc) [inline src/AppBundle/Resources/config/graphql/Realm.types.yml:12]: Let's be consistent w/ periods. I prefer to have periods btw :)

**Comment by austenmc** [comment]
Change the composer.json stuff but go ahead and commit this. I dont like  Real { coreVersion, crateVersion } b/c it's not extendable to learn more about the modules that are installed, which would also need versions, but for right now this is fine.

I think in the future we want something like a configuration connection (an array of configurations):

```
Realm {
  configuration {
    library # like lotgd/core
    name # like "Legend of the Green Dragon Core"
    description # some text here
    version # version string
  }
}
```

File: app/config/config.yml (hunk 1 above)
Comment (Vassyli) [inline app/config/config.yml:7]: Not sure what you mean. This is the standard file after creating the project. I've put some default variables to parameters.yml, but I'm not using symfonys doctrine service anyway, lotgd\core manages the database for us. Real database connection parameters are in .env (which I need to read out manually on windows).

But yeah, configuration is an issue for itself.

File: composer.json (hunk 2 above)
Comment (Vassyli) [inline composer.json]: I had to use this as long as my PR on lotgd/core was hanging. I'll be changing it to the normal github repository now.

**Comment by Vassyli** [comment]
Yeah, I just wanted to have something running and working for a first commit to see what we need to change in the core :)
I'll fix and push and "merge".

**Pull request merged**